src/modules/terminalfont.c

```diff
@@ -69,6 +67,47 @@ static void printXFCE4Terminal(FFinstance* instance)
     printTerminalFont(instance, font);
 }
 
+static void printTilixTerminal(FFinstance* instance)
+{
+    const char* fontName = NULL;
+    const char* defaultProfile = NULL;
+
+    defaultProfile = ffSettingsGetGsettings(instance, "com.gexperts.Tilix.ProfilesList", NULL, "default", FF_VARIANT_TYPE_STRING).strValue;
+
+    if (!defaultProfile)
+    {
+        ffPrintError(instance, FF_TERMFONT_MODULE_NAME, 0, &instance->config.termFontKey, &instance->config.termFontFormat, FF_TERMFONT_NUM_FORMAT_ARGS, "Couldn't find \"Default\" profile in Tilix settings");
+        return;
+    }
+
+    FFstrbuf key;
+    ffStrbufInitAS(&key, 64, "/com/gexperts/Tilix/profiles/");
+    ffStrbufAppendS(&key, defaultProfile);
+    int keyLen = key.length;
```

Comment:
`key.length` is a `uint32_t`. We should keep this to prevent casting errors.

Reply:
Wonder why there is no warning about stuff like this?

Reply:
I didn't enable -Wconversion on the standard build, because there are situations there are situations where it is needed to do a conversion that the c standard doesn't like.
For example, ISO c forbids a conversion from a void pointer to a function pointer, however POSIX requires us to do exactly this in functions like [dlysm](https://linux.die.net/man/3/dlsym)...
To build with all warnings, run `tests/performance.sh`, but don't be shocked by all the warnings you will get, most of them are allowed by POSIX and just not by ISO c, and therefore allowed in a linux only context.
I decided to do this, because i don't want users to see warnings when building the project (in example from the AUR).